docs/src/main/sphinx/release/release-357.md

```diff
@@ -0,0 +1,72 @@
+# Release 357 (XX May 2021)
+
+## General
+
+* Add support for subquery expressions that return multiple columns. E.g., `SELECT x = (VALUES (1, 'a'))`. ({issue}`7773`, {issue}`7863`)
+* Allow aggregation pushdown when `COUNT(1)` is used with `GROUP BY`. ({issue}`7251`)
+* Add support for `CURRENT_CATALOG` and `CURRENT_SCHEMA`. ({issue}`7824`)
```

Comment:
What do they return when current catalog or schema is not set?

Reply:
null